Gems/Atom/RHI/Code/Include/Atom/RHI/ResourceView.h

```diff
@@ -48,10 +52,12 @@ namespace AZ
             ResultCode Init(const Resource& resource);
 
         private:
-            //////////////////////////////////////////////////////////////////////////
-            // RHI::Object
-            void Shutdown() override final;
-            //////////////////////////////////////////////////////////////////////////
+            ///////////////////////////////////////////////////////////////////
+            // IntrusivePtrCountPolicy template overrides
+            void release() const final;
```

Comment:
Add a comment explaining why we are overriding release as oppose to shutdown which is the case everywhere else.

Reply:
Done.